helpers/middleman/menu.rb

```diff
@@ -264,6 +264,13 @@ def get_resources_menu(locale_obj: current_locale_obj)
                   label: 'Webinars & Events',
                   title: locale_page(page: 'events', locale_obj: locale_obj)[:page_title],
                   href: localize_url('events', locale_id: locale_obj[:id]),
+                },
+                {
+                  icon: 'users',
+                  icon_hover: 'users-fill',
+                  label: 'Community Forum',
+                  title: 'Community Forum',
```

Comment:
I believe this is redundant…you just need `label` and not `title`.  I could be wrong (and maybe that's too magical to exist in a Ruby ecosystem with zero way to know that unless you look at the `.html.erb`).

Reply:
Works without `title` 👍

Reply:
Ah @kylorhall  - `title` is for the HTML `title` attribute 🤦‍♂️

Reply:
Yeah, I figured this would do like `title: title || label`…could be wrong (doesn't matter) 👍